console/war/src/main/java/nl/nn/adapterframework/runner/ConsoleFrontendBean.java

```diff
@@ -43,6 +48,27 @@ public class ConsoleFrontendBean implements ApplicationContextAware {
 
 	private @Setter ApplicationContext applicationContext;
 
+	@Autowired
+	private Environment environment;
+
+	private String getFrontendLocation() {
+		String frontendFolder = "classpath:/META-INF/resources/iaf/gui/";
+		if(Arrays.asList(environment.getActiveProfiles()).contains("dev")){
+			String devFrontendLocation = environment.getProperty("frontend.resources.location");
+			if(devFrontendLocation == null) {
```

Comment:
Ik zou isNotBlank gebruikt (StringUtils3) maar voor de rest prima zo. Wil het nog wel testen in Eclipse :)